app/assets/javascripts/Components/__tests__/repo_browser_manual_collection_form.test.jsx

```diff
@@ -0,0 +1,72 @@
+import {render, screen, fireEvent} from "@testing-library/react";
+import {ManualCollectionForm} from "../repo_browser";
+
+// workaround needed for using i18n in jest tests, see
+// https://github.com/fnando/i18n/issues/26#issuecomment-1235751777
+jest.mock("i18n-js", () => {
```

Comment:
Ah yes, can you actually move these mocks into one of the Jest setup files (and remove them from existing places in the codebase)?

Reply:
As it turns out, apparently we don't need these at all! Seems that there was already a Jest setup in `jest_env_setup.js` which defines `I18n` for all Jest tests globally. When writing this and previous tests, I never considered it may be implicitly defined so I didn't try without explicitly defining it - should have looked in some other files first, sorry about that! I've gone ahead and removed this setup from all files that use it.